IaC/BareMetal/fluentbit/README.md

```diff
@@ -0,0 +1,119 @@
+# Debugging Fluent Bit
+
```

Comment:
I think we should split this into two readme files

One for stdout

One for Log Analytics
- include the LA setup there instead of main instructions (for simplicity)

Reply:
The new Fluent Bit - Log Analytics repo uses the LA config.

We could add instructions for using stdout (it's a great debugging tool)